.config/s3.config.php

```diff
@@ -0,0 +1,20 @@
+<?php
+if (getenv('OBJECTSTORE_S3_HOST')) {
+  $CONFIG = array (
+    'objectstore' => array(
+      'class' => '\\OC\\Files\\ObjectStore\\S3',
+      'arguments' => array(
+        'bucket' => getenv('OBJECTSTORE_S3_BUCKET'),
+        'autocreate' => getenv('OBJECTSTORE_S3_AUTOCREATE') ?: true,
```

Comment:
```suggestion
        'autocreate' => getenv('OBJECTSTORE_S3_AUTOCREATE') ?: false,
```

Reply:
We had troubles with the true as it didn't create the bucket.

I don't recall that well, but I think it is better this way.